### ISSUE TITLE
Ajouter le questionnaire de santé de l'ARS

### ISSUE THREAD

**Issue by colinemarie** [hostolab/covidliste#776]
## Contexte / Problème

Une fois sur place, les volontaires doivent remplir le questionnaire de santé officiel de l'ARS, plus des gens l'auront pré-remploi, plus les vaccination peuvent se dérouler rapidement. Doctolib le fournit systématiquement quand un rdv est confirmé par ex.

## Proposition

Ajouter un lien vers le pdf dans le mail de confirmation de match et dans la page de match confirmé.

## Priorité (Requis)

- `P3`: Pas prioritaire

**Comment by mininao** [comment]
Quelques ptites questions :
* Est-ce que ça a été discuté sur slack / est-ce que c'est issu d'un retour d'un centre de vaccination ?
* Est-ce qu'on est sûrs que tous les centres ont le même formulaire, et que donc ça sert d'arriver avec son questionnaire ? Dans mon centre ça avait l'air un peu custom, et peut-être que les centres veulent garder la liberté d'organiser leur process.
* J'ai pas le souvenir d'avoir reçu ce formulaire de la part de doctolib pour ma première injection donc je suis pas sûr ?

**Comment by colinemarie** [comment]
Non, c'est une idée originale de @carsso après avoir rendu visite au centre de Dijon. J'ai regardé en ligne, la majorité des ARS proposent les même questionnaires, mais customent le style en effet...
En fait, via Doctolib ils ont la possibilité de le proposer ce qui arrangeraient surement certains. Tu as raison, je lance la discussion en ligne 🙂

**Comment by mininao** [comment]
([lien du thread slack pour accès facile](https://covidliste.slack.com/archives/C01TV9NDY5V/p1621209094052600))
